### ISSUE TITLE
[Assets] Do not use public asset URL in admin UI for previews

### ISSUE THREAD

**Issue by brusch** [pimcore/pimcore#3301]
Within the admin UI, assets should always delivered through an action, to ensure only authorized persons can access the asset if there's an access restriction (eg. using mod_rewrite) in place. 

- Image should be already covered (need recheck) 
- Documents (PDF): to be done
- Videos: to be done

**Comment by brusch** [comment]
- Documents resolved by ac633358bcca7fe0be3e545ebb6e8652160cd47a

**Comment by nephews** [comment]
If one would like to restrict public access to all assets would it be fine to put this in .htaccess?
(Same as in pimcore manual but without the 'protected' folder)
```
RewriteRule ^/.* - [F,L]
RewriteRule ^var/.*/(.*) - [F,L]
RewriteRule ^cache-buster\-[\d]+/(.*) - [F,L]
```
It seems to work well (pimcore version 5.6), admin works, REST and PHP API seem to work as expected. Are there known limitations/drawbacks?
Thanks in advance
